src/limit-line-length/limit-line-length.css

```diff
@@ -0,0 +1,21 @@
+.diff-content-container.refract-container {
+    position: relative;
+}
+
+.refract-content-container::before {
+    position: absolute;
+    top: 0;
+    height: 100%;
+    border-right: 1px solid #c0c0c0;
+    pointer-events: none;
+    color: transparent;
+    user-select: none;
```

Comment:
since it was in `pre.source` maybe that's why some people reported not being able to select words in diff